kuma/static/styles/components/home/callouts/_layout-debugging.scss

```diff
@@ -0,0 +1,73 @@
+/*
+Layout debugging callout
+-------------------------------------------------------------- */
+
+.callout-layout-debugging {
+    box-sizing: border-box;
+    background-color: #d6f4ff;
+    border: 1px solid #999;
+    box-shadow: 2px 2px 5px #edecec, -2px -1px 5px #edecec;
+
+    @media screen and (min-width: 83.9375em) {
+        background: #d6f4ff url($path-to-images+'embed/promos/picassofox.svg')
```

Comment:
> isn't there a danger that translations of the text will result in larger strings that overlap the fox?

Yes, but as this will not be translated, I focused on ensuring it does not happen with the English copy.